test/unit/apipie/command_test.rb

```diff
@@ -154,4 +156,31 @@ class CommandC < CommandA
 
   end
 
+  context "defaults" do
+
+    before :each do
+      HammerCLI::Settings.load(:global_defaults => {:organization_id => 3})
+    end
+
```

Comment:
These tests needs some refinements but I just wanted to check we are on the right track and didn't want to waist a lot of time.